apps/condo/domains/common/components/PostMessageProvider/globalHandlers.ts

```diff
@@ -0,0 +1,138 @@
+import { useCallback, useContext } from 'react'
+import { notification } from 'antd'
+import { v4 as uuidV4 } from 'uuid'
+import get from 'lodash/get'
+import pickBy from 'lodash/pickBy'
+import dayjs from 'dayjs'
+import { useIntl } from '@open-condo/next/intl'
+import { useAuth } from '@open-condo/next/auth'
+import { useOrganization } from '@open-condo/next/organization'
+import { STAFF } from '@condo/domains/user/constants/common'
+import { useMiniappTaskUIInterface } from '@condo/domains/common/hooks/useMiniappTaskUIInterface'
+import { TASK_STATUS, TasksContext } from '@condo/domains/common/components/tasks'
+import type { RequestHandler } from './types'
+
+export const handleNotification: RequestHandler<'CondoWebAppShowNotification'> = (params) => {
+    const { type, ...restParams } = params
+    notification[type](restParams)
+    return { success: true }
+}
+
+export const useLaunchParamsHandler: () => RequestHandler<'CondoWebAppGetLaunchParams'> = () => {
+    const { locale } = useIntl()
+    const { user } = useAuth()
+    const { organization } = useOrganization()
+    const userId = get(user, 'id', null)
+    const organizationId = get(organization, 'id', null)
+    return useCallback(() => {
+        return {
+            condoUserId: userId,
+            condoUserType: STAFF,
```

Comment:
Why not `user.type`? For now, it's `stuff`, yes. But in future its possible place for the bug.

Reply:
It's a like platform-dependent const, and change like this will include global product decision.

So, we'll need to change Bridge API -> which will cause changing this place to change.

For now const is enough here